src/components/WorldMap.js

```diff
@@ -1,12 +1,120 @@
 import "./styles/WorldMap.css";
 import { Link } from "react-router-dom";
+import { useState, useEffect } from "react";
+import ReactMapGl, { Marker } from "react-map-gl";
 
 export default function WorldMap() {
+  const [countries, setCountries] = useState([]);
+  const [visitedCountries, setVisitedCountries] = useState(Number(0));
+  const [filterInputValue, setFilterInputValue] = useState("");
+  const [markers, setMarkers] = useState([]);
+  const [viewPort, setViewPort] = useState({
+    latitude: 42.123,
+    longitude: 10.123,
+    width: "520px",
+    height: "370px",
+    zoom: 0,
+  });
+
+  const filteredCountries =
+    countries.length > 0 &&
+    countries.filter((country) => {
+      return country.name
+        .toLowerCase()
+        .includes(filterInputValue.toLowerCase());
+    });
+  console.log(filteredCountries);
```

Comment:
You can remove this `console.log`
It's a good practice to remove `console.log` or similar from the code before merging.